blivet/devices/lvm.py

```diff
@@ -1786,8 +1786,9 @@ def __init__(self, cached_lv, size=None, md_size=None, exists=False, fast_pvs=No
 
     @property
     def size(self):
-        if self.exists:
-            return self.stats.size
+        stats = self.stats
```

Comment:
Why use an intermediate variable here? It isn't unreadable to just say if self.stats and return self.stats.size